ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -450,5 +450,5 @@ Outputs:
     Value: !Ref 'InstanceRole'
   IAMInstanceProfile:
     Value: !Ref 'InstanceProfile'
-  ConnectionInstructions:
```

Comment:
In my opinion, we should leave the connection instructions output with some explanation of how to connect, as the ConnectionURI is not self-explanatory in all cases. 

For Notebooks, for example, we will change the ConnectionURI to an ALB DNS record at some point, for for our first release it will be an internal IP addr, and we need a way to point people to a VPN route and to which credentials to use.

Reply:
How about add it back in when you have the docs for it?

Reply:
It is ok to remove ConnectionInstructions from the notebook template iff you add a separate property for starting a browser connection to the notebook server. You could name it something like `NotebookServerUri`, as discussed at standup.